### PR TITLE
tests: extend virt-handler canary upgrade test timeout

### PR DESCRIPTION
**Pull request by acardace**
On some clusters with >3 nodes 120 seconds seem to be not enough.

Signed-off-by: Antonio Cardace <acardace@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/canary_upgrade_test.go [hunk 1]
```diff
@@ -187,14 +195,15 @@ var _ = Describe("[Serial][sig-operator]virt-handler canary upgrade", func() {
 			},
 		})
 
-		err := updateVirtHandler()
+		err = updateVirtHandler()
 		Expect(err).ToNot(HaveOccurred())
 
+		updateTimeout := time.Duration(canaryTestNodeTimeout * nodesToUpdate)
```
Comment (xpivarc) [inline tests/canary_upgrade_test.go:201]: Do you need to cast it `Duration`?

**Review by xpivarc — APPROVED** [review]
Thanks for the PR!

File: tests/canary_upgrade_test.go (hunk 1 above)
Comment (acardace) [reply tests/canary_upgrade_test.go:201]: vim-go complains a lot if I don't, so it does seem necessary.

**Review by dhiller — COMMENTED** [review]
/approve

**Pull request merged**